xbmc/cores/VideoPlayer/DVDCodecs/Video/DXVA.cpp

```diff
@@ -540,12 +575,14 @@ bool CContext::CreateSurfaces(const D3D11_VIDEO_DECODER_DESC& format, uint32_t c
     texDesc.MiscFlags |= D3D11_RESOURCE_MISC_SHARED;
   }
 
-  CLog::Log(LOGDEBUG, "DXVA: allocating {} surfaces with format {}.", count, format.OutputFormat);
+  CLog::Log(LOGDEBUG, "DXVA: allocating {} surfaces with format {} {}.", count, format.OutputFormat,
+            DX::DXGIFormatToString(format.OutputFormat));
```

Comment:
Maybe better drop format number from log (string name only)

Reply:
yes I was thinking about debugging in VS but after checking there is no need.
I'll do the other ones as well.